releasenotes/notes/fix-aws-lambda-infinite-recursion-crash-2cba8a190411cc7e.yaml

```
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    aws_lambda: Fixes a bug in which signals were wrapped infinitely by adding a condition and removing
+    signals after an invocation.
```

minor nit: Can we exclude implementation details from the release note and instead highlight customer impact.

```suggestion
    aws_lambda: Fixes a ``RecursionError`` which is raised when aws lambda signal handlers are wrapped infinitely. This caused lambdas to crash on startup.
```